sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/implementation/LeaseStoreManagerImpl.java

```diff
@@ -435,16 +435,15 @@ public LeaseStoreManagerBuilderDefinition hostName(String hostName) {
             this.requestOptionsFactory.createFeedOptions());
 
         return query.flatMap( documentFeedResponse -> Flux.fromIterable(documentFeedResponse.results()))
-            .map( ServiceItemLease::fromDocument);
+            .map(ServiceItemLease::fromDocument);
     }
 
     private String getDocumentId(String leaseToken)
     {
         return this.getPartitionLeasePrefix() + leaseToken;
     }
 
-    private String getPartitionLeasePrefix()
-    {
+    private String getPartitionLeasePrefix() {
         return this.settings.getContainerNamePrefix() + "..";
```

Comment:
we should consider making these as defined constants.

Reply:
changed to a constant